### DISCUSSION TITLE
Colours

### DISCUSSION THREAD

**Discussion by Nimril** [elvirbrk/NoteHighlight2016#145] [Ideas]
I'm hoping this is the right place to ask/request this.
I'm using NoteHighlight to help me make notes in OneNote while trying to learn Python. The plethora of style/colour options are vast, and I've just gone through each style to try and find a style that helps. There are some crazy combinations. However, for me, a style that matches IntelliJ's Darcular default colours would help. So I'm looking at the same colours when looking at the notes that I would normally see when trying to learn Python using IntelliJ.

**Comment by Dridou** [comment]
Hi everyone,

I just tested ALL the themes too see which one suits the best.
I am in Dark mode and here are the themes which suits "ok" and their preview with names at the bottom:

![image](https://user-images.githubusercontent.com/22415102/213745580-84019b11-a2dc-4ff4-8e20-9a4617e6a62d.png)
![image](https://user-images.githubusercontent.com/22415102/213745641-82402b81-3ebc-4775-adcf-01942a398f2a.png)
![image](https://user-images.githubusercontent.com/22415102/213745674-49dd050e-f81e-4700-8152-84479eb423d3.png)
![image](https://user-images.githubusercontent.com/22415102/213745707-59e4eee9-dfce-47d9-a234-8b40fb50bf21.png)
![image](https://user-images.githubusercontent.com/22415102/213745788-1f389e4c-aef4-4962-b483-4b09f0316d9a.png)
![image](https://user-images.githubusercontent.com/22415102/213745824-5e84b2a0-dc87-40fa-bd41-1bb74151f880.png)

I think I'll choose github / bdclear ones.

Hope this helps you.
